.github/workflows/publish-module-api-doc.yaml

```diff
@@ -1,4 +1,4 @@
-name: Publish module API documentation
+name: Build and publish module API documentation
```

Comment:
```yaml
on:
  push:
    branches: [master]
  pull_request:
    types: [opened, reopened, synchronize, labeled]
  workflow_dispatch:
```

`labeled` is redundant here: we don't want to re-run the workflow at changes in the pull request labels. It was relevant, when the `full-ci` logic was here, but it is removed now.

If omit `types`, [it'll be](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) `[opened, reopened, synchronize]` -- fits good for us.

I think the overall logic would be pretty similar to one of existing run-always workflows, say, `debug.yml`.

Reply:
Updated:

```diff
--- a/.github/workflows/publish-module-api-doc.yaml
+++ b/.github/workflows/publish-module-api-doc.yaml
@@ -1,10 +1,10 @@
-name: Publish module API documentation
+name: Build and publish module API documentation
 
 on:
   push:
     branches: [master]
   pull_request:
-    types: [opened, reopened, synchronize, labeled]
+    types: [opened, reopened, synchronize]
   workflow_dispatch:
 
```